utils/string.test.ts

```diff
@@ -142,7 +142,6 @@ describe('String Utilities', () => {
       expect(utils.humanReadableBytes(1234567890)).toBe('1.23 GB');
       expect(utils.humanReadableBytes(1234567890123)).toBe('1.23 TB');
       expect(utils.humanReadableBytes(1234567890123456)).toBe('1.23 PB');
-      expect(utils.humanReadableBytes(1234567890123456789)).toBe('1.23 EB');
```

Comment:
not sure how we'd have this test with the following so I just removed it.

> In JS, Numbers are stored as double-precision floating-point numbers according to the [IEEE 754 standard](https://en.wikipedia.org/wiki/IEEE_754). Because of this, numbers can only retain accuracy up to a certain amount of digits. If the programmer enters additional digits, those digits will be lost in the conversion to the Number type and will result in unexpected behavior.

Reply:
we could ignore the eslint complaint as well